client/components/PubEdgeListing/pubEdgeListing.scss

```diff
@@ -1,6 +1,7 @@
 .pub-edge-listing-component {
 	display: flex;
 	flex-direction: column;
+	padding: 20px 0;
```

Comment:
If this is in place to provide the correct amount of distance from other Pub body elements, I think I'd prefer to address them individually by their class names given in `PubDocument.js` and referenced in `pubDocument.scss`. I don't think this component ought to BYOP, so to speak. At any rate I'm concerned tha the combined effect of this padding plus the margin there is maybe too much.